web/src/helpers/server/get-first-app/graphql/getAppsByUserAndTeams.graphql

```diff
@@ -0,0 +1,13 @@
+query AppsByUserAndTeam($user_id: String!, $team_id: String!) {
```

Comment:
same as with the default team, why not limit to `1` to improve performance, this query may actually become large

Reply:
Done